Controller/RouterController.php

```diff
@@ -58,4 +60,30 @@ public function asset(Request $request): Response
 
         return $this->processor->getResponse($request, $data['hash'], $data['config'], $data['filename']);
     }
+
+    public function makeResponse(Request $request): Response
+    {
+        $result = $this->handler->handle($request);
+        $json = $this->templating->render($result['template'], $result['context']);
+
+        $data = \json_decode($json, true);
+
+        if ($data === false) {
+            throw new \RuntimeException('Failed to decode JSON');
+        }
+
+        $response = new Response();
+
+        $response->setContent($data['content'] ?? null);
```

Comment:
If content is missing you should raise an error with a message explaining that a json with a mandatory content field is required